.github/workflows/ci-test-ginkgo.yml

```diff
@@ -84,6 +91,12 @@ jobs:
 
       - name: Run KubeArmor
         run: |
+          if [[ ${{ matrix.runtime }} == "containerd" ]]; then
```

Comment:
Why another if statement? Is there any problem for putting in the same if statement?

Reply:
> Why another if statement? Is there any problem for putting in the same if statement?
I thought it'd make it a bit more readable(as the kubearmor related images would be in a different block and the multiubuntu docker save block would be prominent enough. I'll make that change right away :)